cnxepub/tests/test_formatters.py

```diff
@@ -433,6 +433,10 @@ def test_binder(self):
             expected_content = f.read()
 
         actual = str(SingleHTMLFormatter(self.desserts))
+        # Uncomment this if/when you get a test failure
```

Comment:
Hmm, that works, but makes the test less efficient.  One of the other tests does do that, over in archive, I think). o.k. let me see what I can cook up. Ohh, can I make it conditional on the test passing? Then there's no commenting to do!